### PR TITLE
LND Channel backup. Bug fixed. Motivated by Issue #958

### LINKED ISSUE #958: Is “/mnt/static-channel-backup-external” listed?  nope 8-(
Lightning/Channel Backup/Option 1: Local Backup/Set up a mounting point for the storage device

$ sudo mkdir /mnt/static-channel-backup-external
$ sudo chattr +i /mnt/static-channel-backup-external

done

8marchprob!

8marprob2

I ran through this part of the guide twice, 
updated the /etc/fstab twice

Is “/mnt/static-channel-backup-external” listed?  
nope "...no such file or directory"
 
Any idea why it isnt listed?

in the meantime, I'll try using the remote github repositry method to backup.
cheers,

### Environment
 - Hardware platform: [e.g. Raspberry Pi 4, 
 - Operating system: [e.g, Raspberry Pi OS 64bit, Debian 11, ... ]

### LINKED ISSUE #981: [BUG] Wrong path for the target of "df" in the SCB backup guide
### Describe the bug
Bug reported here by Telegram user @El_Stone: [https://t.me/raspibolt/9325](https://t.me/raspibolt/9325)

In the SCB backup guide, the command `df -h static-channel-backup-external` returns an error message `df: static-channel-backup-external: No such file or directory`.

### Location in guide
At the end of the following section: https://raspibolt.org/guide/lightning/channel-backup.html#formatting

### Expected behavior
The command should output the expected information.

### Proposed fix
Use the absolute path of the folder: `df -h /mnt/static-channel-backup-external`

### PR DESCRIPTION
**Pull request by mktech-gh**
#### What

Based on the implemented backup option with Dropbox, I wanted to switch the backup to the new variants. At the same time I also wanted to make a pull request to fix issue #958. I found a few more small bugs and fixed them.

Fixes #958 
Fixes #981

So I got the following error messages because the file channel.backup exists with a size of 0 bytes and root access rights:

admin@raspibolt:~ $ ls -al ~/.lnd/data/chain/bitcoin/mainnet/channel.backup
-rw-r--r-- 1 root root 0 Feb  1 09:58 /home/admin/.lnd/data/chain/bitcoin/mainnet/channel.backup

......
Apr 10 17:29:04 raspibolt lnd[114726]: 2022-04-10 17:29:04.547 [ERR] LTND: Shutting down because error in main method: unable to start server: unable to refresh backup file: unable to extract on disk encrypted SCB: payload size too small, must be at least 24 bytes
Apr 10 17:29:04 raspibolt lnd[114726]: 2022-04-10 17:29:04.548 [INF] RPCS: Stopping RPC Server
......
Apr 10 17:29:26 raspibolt systemd[1]: Stopped LND Lightning Network Daemon.
Apr 10 17:29:26 raspibolt systemd[1]: lnd.service: Consumed 14.296s CPU time.

### Why

The imprecise path information is not easy to handle for users without Commandline knowledge.

#### How

I changed the text ;-)

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] simple bug fix

Fixes #958

#### Test & maintenance

I implemented the installation according to the corrected guide

#### Animated GIF (optional)

Add some snazz if you feel like it :)

A fulfilling job

### CODE REVIEW

**Review by VajraOfIndra — APPROVED** [review]
Great bug fixes, thank you! :)
LGTM, utACK

My only comment is regarding the new section concerning the 0 bytes channel.backup file... I'm not sure I understand what the issue really is and how a edge case or common it is. 

Also, should we explain where the file size is indicated in the output of ls -la and what unit it is?

@RooSoft @Stadicus any insights or comments on this 0-byte backup issue?

**Comment by mktech-gh** [comment]
I did some research on the 0Byte channel.backup file.

 Based on my recording, I see that I tested the backup to Dropbox at the beginning of February (the 0 byte file was created on February 1st).  
For the test, I changed the channel.backup file with "touch /data/lnd-backup/channel.backup" to trigger a backup manually.  
It's possible that I accidentally created the ~/.lnd/data/chain/bitcoin/mainnet/channel.backup file as well.

 So I think the problem was caused by myself.

 I suggest I delete my addendum and make another pull request.

 Is this procedure correct?

 Sorry for the circumstances.

**Comment by VajraOfIndra** [comment]
Hi @mktech-gh!

Rather than closing this PR and opening a new one without your addendum, you could simply remove it from your branch. The PR will then show the updated version.

Also, it seems you added a change for the charge-lnd bonus guide in this PR. It should belong to a separate new PR. The way to do it would be to create a new branch in your repository (e.g. a branch named "charge-lnd-bug-fix"), make your changes in that new branch and then create a PR from it.

Basically, rather than working from your master branch (mktech-gh/master), you could create a new branch for each PR you'd like to make. Let me know if that makes sense or not.

**Comment by mktech-gh** [comment]
Hello @VajraOfIndra

Thank you for your help 🙏🏻. That's great for me. That way I can constantly improve.

Using the branch now makes sense to me.🤔and i will use it now

I have now deleted my changes regarding the 0 byte file in channel-backup.md.

How can I undo the changes in "charge-lnd bonus guide"? I would like to re-do these changes in a new branch?

**Comment by VajraOfIndra** [comment]
> How can I undo the changes in "charge-lnd bonus guide"? I would like to re-do these changes in a new branch?

There might a much better way (like cancelling commits), but the way I do it usually is either removing the changes I made if I remember exactly what they were or else, I delete the entire content of the file and paste the content of the file from the RaspiBolt master branch (the one being used for the official website), like that I'm sure the file is exactly as it should be and all my changes are gone.

But again this is hacky, I'm sure there is a more porfessional way to do it playing with commits :)

**Comment by mktech-gh** [comment]
OMG: 3 attempted commits ;-). But your hack worked :-)
@VajraOfIndra: My thanks to you.

**Review by VajraOfIndra — APPROVED** [review]
Fixing 3 typos at once, excellent! :)

LGTM, tACK

Let's go to master! :tada:
